article/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.contrib.auth.models import User
+import uuid
+# Create your models here.
+
+class Article(models.Model):
+    id = models.UUIDField(default=uuid.uuid4,  unique=True, primary_key=True, editable=False)
+    user = models.ForeignKey(User,on_delete=models.SET_NULL, null=True, blank=True)
+    content = models.TextField(max_length=10000)
+    tags = models.CharField(max_length=100)
+    created = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return str(self.user.username)
+
+
+class ArticleComment(models.Model):
+    id = models.UUIDField(default=uuid.uuid4,  unique=True, primary_key=True, editable=False)
```

Comment:
primary_key= True means unique and not null True,
id = models.UUIDField(default=uuid.uuid4, primary_key=True, editable=False)

Reply:
Just for more safety purpose